Smalltalk.gitignore

```diff
@@ -1,15 +1,24 @@
 # changes file
 *.changes
+*.chg
 
 # system image
 *.image
+*.img7
+*.img
 
 # Pharo Smalltalk Debug log file
 PharoDebug.log
 
 # Squeak Smalltalk Debug log file
 SqueakDebug.log
 
+# Dolphin Smalltalk source file
+*.sml
```

Comment:
This can help https://github.com/dolphinsmalltalk/Contributions the official contribution repository.